demo/src/components/UserInfoForm/UserInfoForm.js

```diff
@@ -10,7 +10,7 @@ import PropTypes from 'prop-types';
  *      column by itself.
  * @returns {!JSX} A form group with the given label and control.
  */
-const makeFormGroup = (label, control, useWholeRow=true) => {
+export const makeFormGroup = (label, control, useWholeRow=true) => {
```

Comment:
Should be `useWholeRow = true`
https://google.github.io/styleguide/jsguide.html#features-functions-parameter-return-types

Reply:
Added spaces